components/ui/testimonial-card.tsx

```
@@ -0,0 +1,25 @@
+import React from "react";
+
+interface TestimonialCardProps {
+  quote: string;
+  author: string;
+  role: string;
+}
+
+export function TestimonialCard({ quote, author, role }: TestimonialCardProps) {
+  return (
+    <div className="bg-white p-6 rounded-lg shadow-md border border-gray-100">
+      <div className="text-3xl text-purple-200 mb-4">"</div>
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)